### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ddohler**
# Overview
Adds complete (I hope) documentation for how to get a DRIVER instance set up and running.

# Testing instructions
- Read through the new documentation and make sure everything makes sense
- It might not be a bad idea to try to follow the instructions and do a test deployment in the World Bank demo account; with the setup wizard + the cloudformation stack, it should be pretty quick.

### CODE REVIEW

**Comment by ddohler** [comment]
This is ready for review.

**Comment by pcaisse** [comment]
Ultimately I got this working: https://peter-test.roadsafety.io

File: doc/system-administration.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # System administration and troubleshooting
 
-The goal of this document is to provide enough information to allow a system administrator to maintain a production DRIVER system, and to aid in troubleshooting if something goes wrong. This assumes a production system has already been deployed using the default configuration.
```
[the remark this answers is not in the document]
Comment (ddohler) [reply doc/system-administration.md]: The KeyName parameter refers to an existing key pair but doesn't necessarily require that there be an existing EC2 instance. I'll add a link to the AWS documentation on generating key pairs.

File: doc/system-administration.md [hunk 2]
```diff
@@ -25,43 +27,107 @@ Each of these servers also includes:
    * Firewall configuration via [ufw](https://help.ubuntu.com/community/UFW)
    * Logs for each locally running service, see [logging.md](logging.md) for details
 
+## Deployment
+### 1. Initial setup
 
-## Deploying updates
+Using whatever computing resources are available to you, set up three blank servers running Ubuntu 14.04. Amazon Web Services (AWS) is the preferred way to set up these servers, but other cloud providers such as Microsoft Azure should work as well. If you have physical servers provided by your IT department, they will also work.
+
+If you are using AWS, you can take advantage of the CloudFormation template file located at `TODO`. See **CloudFormation** below.
```
[the remark this answers is not in the document]
Comment (ddohler) [reply doc/system-administration.md]: Oops, yes, good catch. My initial writeup of the Cloudformation instructions disappeared into the ether somehow, and I ended up rewriting in a rush.

File: doc/system-administration.md [hunk 3]
```diff
@@ -25,43 +27,107 @@ Each of these servers also includes:
...
+- `local_country_code`: Set this to your country's two-letter code
+- `local_center_lat_lon`: Set this to the location where you want DRIVER's maps to be centered by default.
+
+Don't worry about making a mistake here -- you can always change the parameters and re-run the deployment process (next section) if you realize you need to make changes later.
+
+### 4. Deploy
+
+Once the **group-vars** and **inventory** files are in place, deployment may be performed by opening a terminal, switching to the directory of the DRIVER source code and running the following command (NOTE: Make sure to change the `--user` argument to a user that has sudo privileges on those servers):
 ```
 ansible-playbook -i deployment/ansible/inventory/production --user=ubuntu \
```
[the remark this answers is not in the document]
Comment (ddohler) [reply doc/system-administration.md]: I added instructions to type `yes` if this message appears, and to re-run if that error is encountered.

**Pull request merged**